code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -142,19 +142,22 @@
 
 /datum/species/proc/create_organs(var/mob/living/carbon/human/H) //Handles creation of mob organs.
 
+
+	for(var/obj/item/organ/internal/iorgan in H.internal_organs)
+		if((iorgan in H.internal_organs))
```

Comment:
next question, why's it there at all? You just got iorgan out of H.internal_organs, so it's got to be in there